app.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
```

Comment:
This app.js file is not needed here, the index.js file is already doing the work.